src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotTests.cs

```diff
@@ -161,16 +223,42 @@ private static T GetValue<T>(ITableEntriesSnapshot snapshot, string columnName)
             return (T)content;
         }
 
-        private static IssueMarker CreateMarker(string ruleKey = "rule key") =>
+        private static IssueMarker CreateMarkerWithSpan(string ruleKey = "rule key") =>
             new IssueMarker(CreateIssueViz(new DummyAnalysisIssue { RuleKey = ruleKey }), new SnapshotSpan());
 
-        private static IAnalysisIssueVisualization CreateIssueViz(IAnalysisIssue issue)
+        private static IssueMarker CreateMarkerWithPath(string primaryFilePath, params IAnalysisIssueFlowVisualization[] flowVizs) =>
```

Comment:
`CreateMarkerWithSpan` implies that `CreateMarkerWithPath` doesn't have a span.
Maybe a single `CreateMarker` with default parameters and use named parameters to initialize them in usage?

 `private static IssueMarker CreateMarker(string ruleKey = "rule key", string primaryFilePath = null, params IAnalysisIssueFlowVisualization[] flowVizs)`

Reply:
I knew there was a reason I didn't have a single method; if you writing something like `CreateMarker(primaryFilePath: "foo", CreateFlowViz("bar"));`
... the compiler complains that `primaryFilePath` is used out of position and is followed by an unnamed argument.

I'll rename the methods to `CreateMarker` and `CreateMarkerWIthSpecifcPaths`.

Reply:
You can remove the `params`, and do `CreateMarker(primaryFilePath:"foo", flows: new []{ CreateFlowViz("bar") }`

Reply:
Yes, but that's more cumbersome, particularly since we don't have any tests that need set both the rule key and the paths.